AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -67,6 +67,17 @@ - (BOOL)saveLog:(id<MSLog>)log withGroupId:(NSString *)groupId flags:(MSFlags)fl
                                    targetKey ? [NSString stringWithFormat:@"'%@'", targetKey] : @"NULL", (unsigned int)persistenceFlags];
   }
   return [self executeQueryUsingBlock:^int(void *db) {
+           // Check maximum size.
+           NSArray<NSArray *> *rows = [MSDBStorage executeSelectionQuery:@"PRAGMA max_page_count;" inOpenedDatabase:db];
```

Comment:
You can probably use `[MSDBStorage  getMaxPageCountInOpenedDatabase]`. (Which should not have "get" in its name, we missed that)